administration/src/bp-modules/cards/hooks/useCardGenerator.test.tsx

```diff
@@ -11,19 +11,44 @@ import createCards, { CreateCardsError, CreateCardsResult } from '../../../cards
 import deleteCards from '../../../cards/deleteCards'
 import { DynamicActivationCode, StaticVerificationCode } from '../../../generated/card_pb'
 import { Region } from '../../../generated/graphql'
+import { ProjectConfigProvider } from '../../../project-configs/ProjectConfigContext'
 import bayernConfig from '../../../project-configs/bayern/config'
+import { ProjectConfig } from '../../../project-configs/getProjectConfig'
+import koblenzConfig from '../../../project-configs/koblenz/config'
+import nuernbergConfig from '../../../project-configs/nuernberg/config'
+import showcaseConfig from '../../../project-configs/showcase/config'
 import downloadDataUri from '../../../util/downloadDataUri'
 import { AppToasterProvider } from '../../AppToaster'
 import useCardGenerator from './useCardGenerator'
 
-const wrapper = ({ children }: { children: ReactNode }) => (
-  <MemoryRouter>
-    <AppToasterProvider>
-      <ApolloProvider>{children}</ApolloProvider>
-    </AppToasterProvider>
+const wrapper = ({
```

Comment:
:upside_down_face: Not changed in this PR and not important at all but usually mocks are directly below imports in our projects. I like that for consistency.